src/components/Options/index.tsx

```diff
@@ -155,6 +177,14 @@ export default class Options extends Component<Props, State> {
             onChange={onEncoderOptionsChange}
           />
         }
+
+        <div class={style.sizeDetails}>
+          <GzipSize
+            // @todo: once we have a nice way to pass down the original image
+            // (image size?), pass compareTo prop here to show size delta.
+            data={image.file}
+          />
```

Comment:
Why are we gzipping? It's generally an anti-pattern to gzip images. It adds processing overhead and often, size overhead.

Reply:
+1

Reply:
good point, sadly now that component is useless